packages/web-components/src/accordion/accordion-item/accordion-item.options.ts

```diff
@@ -0,0 +1,9 @@
+/**
+ * An Accordion Item can be medium, large, and larger in height
+ */
+export type AccordionItemSize = 'medium' | 'large' | 'larger';
```

Comment:
In FUIR9, there are 4 sizes (applicable to `AccordionHeader`): `'small' | 'medium' | 'large' | 'extra-large'`